sky/serve/replica_managers.py

```diff
@@ -163,6 +168,34 @@ def _get_resources_ports(task_yaml: str) -> str:
     return task_resources.ports[0]
 
 
+def _get_use_spot_override(task_yaml: str,
+                           resource_override: Optional[Dict[str, Any]]) -> bool:
+    """Get whether the task uses spot
+    from either overeride or task_yaml.
+    """
+    if resource_override is not None:
+        use_spot_override = resource_override.get('use_spot')
+        if use_spot_override is not None:
+            assert isinstance(use_spot_override, bool)
+            return use_spot_override
+    task = sky.Task.from_yaml(task_yaml)
+    assert len(task.resources) == 1, task
```

Comment:
this is not true for multiple acc

Reply:
I think it's fine to assume that multiple resources share the same port.

Reply:
We could have a check in core functions to make sure even if multi acc is used, there is only one ports.